### ISSUE TITLE
Can imagenet-1k-boxes allowed?

### ISSUE THREAD

**Issue by Ice-cool** [eccv22-ood-workshop/ROBIN-dataset#12]
In the Q&A, the ImageNet-1k is allowed in the image classification. What about the boxes-level annotations? (https://image-net.org/download-bboxes.php)
```
Q: Is it allowed to use additional data for training purposes?

A: No, only ImageNet-1k and ImageNet-1k pretrained models can be used for the challenge.
```

**Comment by DTennant** [comment]
No, Only the images and the classification labels from ImageNet-1k can be used.
I will update the QA accordingly.

**Comment by Ice-cool** [comment]
@DTennant Thank you. When will the baseline about object detection release?

**Comment by DTennant** [comment]
Hi,
We are still waiting for the confirmation from our sponsor, the baseline for the three tasks as well as the data processing script and detailed evaluation protocol will be released after everything is confirmed.

**Comment by Ice-cool** [comment]
The annotations are incomplete. For example, there are three chairs in `iid_test/Images/chair/2008_007194.jpg`, but only two chairs are annotated. (there are many similar images in test data)

**Comment by DTennant** [comment]
Hi,
We have released the codalab server for evaluation, for the detection baseline, we aim to release it by the weekend.
